guides/security/device_profiles.md

```diff
@@ -29,3 +31,9 @@ We have agreed a clear business case for people in engineering roles to use an a
  - Evernote
 
  For Mac users, any app downloaded from the App Store is considered safe to install on your device.
+ 
+ # Related Standards, Policies and Processes
+ - AUP Policy
```

Comment:
Not sure if these are meant to be links or just words - fine either way just wanted to draw attention if it wasn't intentional